include/rotatingMHD/time_discretization.h

```diff
@@ -222,6 +259,20 @@ class VSIMEXMethod : public DiscreteTime
   */
   void set_desired_next_step_size(const double time_step_size);
 
+  template<typename DataType>
+  DataType extrapolate(const DataType &old_values,
+                       const DataType &old_old_values) const;
+
+  template<typename DataType>
+  void extrapolate(const DataType &old_values,
+                   const DataType &old_old_values,
+                   DataType       &extrapolated_values) const;
+
+  template<typename DataType>
+  void extrapolate_list(const std::vector<DataType> &old_values,
+                        const std::vector<DataType> &old_old_values,
+                        std::vector<DataType>       &extrapolated_values) const;
```

Comment:
There are implemented but never used. Did you test them?

Reply:
No not yet but I intend to use them for the extrapolation inside the assembly methods.